src/utils.jl

```diff
@@ -13,6 +13,35 @@ function escapeHTML(i::String)
     return o
 end
 
+macro retry(expr)
+    :(@retry 2 $(esc(expr)))
+end
+
+macro retry(N, expr)
+    :(@retryif Any $N $(esc(expr)))
+end
+
+macro retryif(cond, expr)
+    :(@retryif $(esc(cond)) 2 $(esc(expr)))
+end
+
+macro retryif(cond, N, expr)
```

Comment:
Hi @quinnj 
I'm a fan of macros to make it harder to get retries catch wrong!
Do you have any interest in making a PR against [Retry.jl](https://github.com/samoconnor/Retry.jl) to add these retry macros there?
It would be a good thing to have more consistent utilities for this stuff across packages.
(Some day some of this should be in `Base` to encourage everyone to do error handling better).
I'd be happy to move `Retry.jl` to JuliaWeb to make collaboration easier.